### PR TITLE
SDK-4228 and SDK-3915

### PR DESCRIPTION
**Pull request by wiperz1789**
Contains support for emotion-sdk and also supports blink, blink-rate and neutral

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -75,13 +75,21 @@ endif( DEFINED OpenCV_DIR )
 # look for the Boost includes / libraries:
 #         Boost_FOUND, Boost_INCLUDE_DIRS, Boost_LIBRARY_DIRS, Boost_LIBRARIES,Boost_VERSION
 set(Boost_USE_MULTITHREADED ON)
+
+if (MSVC)
```
Comment (adennie) [inline CMakeLists.txt:79]: * don't need to specify `Boost_DEBUG OFF`, that's off by default.
* `Boost_COMPILER` is specified twice below

**Review by adennie — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (wiperz1789) [reply CMakeLists.txt:79]: @adennie  I added `Boost_DEBUG ON` to test boost library issue. turned off later, will remove it.

File: CMakeLists.txt [hunk 2]
```diff
@@ -101,16 +107,50 @@ set (AFFECTIVA_FOUND FALSE)
...
+        find_file(AFFECTIVA_DLL_RELEASE
+              NAMES affectiva-vision.dll
+              HINTS ${AFFECTIVA_SDK_DIR}/bin/release)
+        find_file(AFFECTIVA_DLL_DEBUG
+              NAMES affectiva-vision.dll
+              HINTS ${AFFECTIVA_SDK_DIR}/bin/debug)
+
+        find_package_handle_standard_args(AFFECTIVA DEFAULT_MSG AFFECTIVA_DLL_DEBUG)
+        find_package_handle_standard_args(AFFECTIVA DEFAULT_MSG AFFECTIVA_DLL_RELEASE)
+
```
Comment (adennie) [inline CMakeLists.txt:136]: I don't think we need this commented out logic below.  Let's remove it so it doesn't get merged in.

**Review by adennie — APPROVED** [review]
Looks good, just wanted to remove some commented out logic.  OK to to merge after that.

**Review by umangmehta12 — APPROVED** [review]
LGTM apart from Andy's comment

File: CMakeLists.txt (hunk 2 above)
Comment (wiperz1789) [reply CMakeLists.txt:136]: fixed

**Pull request merged**